Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelSelectTypePage.xaml

```diff
@@ -50,14 +50,14 @@
             <UserControls:ucButton x:Name="xBrowseLoadButton" ButtonType="RoundTextAndImageButton" ButtonText="Load" ButtonImageType="MoveDown" ButtonFontImageSize="15" ButtonImageWidth="15" ButtonImageHeight="15" Visibility="Collapsed"  Click="BrowseButtonClicked" ButtonStyle="{StaticResource $RoundTextAndImageButtonStyle}"/>
             <UserControls:ucButton x:Name="xPreviewButton" ButtonType="RoundTextAndImageButton" ButtonText="Preview" ButtonImageType="Search" ButtonFontImageSize="15" ButtonImageWidth="15" ButtonImageHeight="15" Visibility="Collapsed" Click="PreviewButtonClicked" ButtonStyle="{StaticResource $RoundTextAndImageButtonStyle}"/>
 
-            </DockPanel>
+        </DockPanel>
 
         <Label Name="SourceRviewLable" Grid.Row="4" Grid.Column="0" HorizontalAlignment="Left" Visibility="Collapsed" Content="Source Preview:" VerticalAlignment="Top"  />
 
         <usercontrols:XMLViewer x:Name="XMLViewer" Grid.Row="4" Grid.Column="1"  Visibility="Collapsed" Margin="0,0,5,0"/>
 
         <Label Name="XMLTemplatesLable" Grid.Row="4" Grid.Column="0" HorizontalAlignment="Left" Visibility="Collapsed" Content="Request Templates:" VerticalAlignment="Top"  />
         <ginger:ucGrid x:Name="XMLTemplatesGrid" Grid.Row="4" Grid.Column="1" Visibility="Collapsed" Title="Request Templates" ShowRefresh="Collapsed" ShowEdit="Collapsed"  ShowAdd="Visible" ShowDelete="Visible" ShowClearAll="Visible" ShowUndo="Collapsed" EnableTagsPanel="False" ShowUpDown="Collapsed"></ginger:ucGrid>
-
+        <Label Name="YamlNote" Grid.Row="4" VerticalAlignment="Top" Visibility="Collapsed" FontFamily="Roboto" HorizontalAlignment="Left" Height="44" Margin="10,274,0,0" Width="500" Grid.ColumnSpan="2">Note: Swagger Open API 3.0 is supported (YAML and Json)</Label>
```

Comment:
Change it to 'Note: Swagger 2.0 and Swagger Open API 3.0 is supported (YAML and Json)'